api/server/graph.cc

```diff
@@ -463,26 +463,40 @@ std::string Graph::nic_add(const app::Nic &nic) {
         return "";
     }
 
-    // Link branch (inside)
-    if (pg_brick_link(gn.firewall.get(),
-                      gn.antispoof.get(), &app::pg_error) < 0) {
-        PG_ERROR_(app::pg_error);
+    // Build branch and set head
+    if (nic.bypass_filtering) {
```

Comment:
you coukd have write:
if (!nic.bypass_filtering) {
...
} else if (app::config.packet_trace) {
...
} else {
...
}

same result with less lines of code